src/common/base/abstractQuick.ts

```diff
@@ -149,7 +149,6 @@ export default abstract class QuickPromotionCommand<T extends typeof SfCommand>
    */
   private buildPromoteOptions(deployId: string): void {
     this.deployOptions = {
-      testLevel: 'Default',
```

Comment:
This is the main change: we don't need to send testLevel on a quick promotion request.

Reply:
The testLevel is optional right ?

Reply:
We don't need to specify a test level on a quick deploy since it only deploy metadata, it does't run the tests. They were run in the previous validate-only deployment.